lib/engines/content_object_store/app/controllers/content_object_store/content_block/editions/steps_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ContentObjectStore::ContentBlock::Editions::StepsController < ContentObjectStore::BaseController
```

Comment:
Could we be more specific in naming this controller? This are the steps to add a new Edition to an existing Document?

Reply:
Maybe it's a workflow actually? Then we can combine the steps and the workflow in one controller? Then we have a workflow as follows for an initial edition:

- `review` (move this from the EditionsController)
- `publish`

And for a new edition on an existing document:

- `confirm_links`
- `schedule_or_publish`

I'll give it a go 👍